src/query_parser/v2/parser.y

```diff
@@ -571,6 +572,17 @@ text_expr(A) ::= QUOTE ATTRIBUTE(B) QUOTE. [TERMLIST] {
   A->opts.flags |= QueryNode_Verbatim;
 }
 
+text_expr(A) ::= SQUOTE ATTRIBUTE(B) SQUOTE. [TERMLIST] {
```

Comment:
Could you add tests with verbatim escaping '*' and '?', I think that we do not support that.